frontend/mono_frontend/src/pages/CardsList/CardsList.jsx

```diff
@@ -1,43 +1,82 @@
-import { useEffect, useState } from 'react'
+import { Link, useRouteLoaderData } from 'react-router-dom'
+import { checkAuthLoader } from '../../utils/auth'
 import { BACKEND_URL } from '../../config/envs'
 import './CardsList.css'
-import { CardItem } from '../../components/CardItem'
-import { useRouteLoaderData } from 'react-router-dom'
+import PopUpManager from '../../components/PopUpManager'
+import { convertToMoneyFormat } from '../../utils/convertToMoneyFormat'
+import { CardIcon } from '../../components/CardIcon'
 
-export const getCards = async function (token) {
+export const getCards = async function () {
 	const endpoint = `${BACKEND_URL}/monobank/monocards`
+	const token = await checkAuthLoader()
+	try {
+		const response = await fetch(endpoint, {
+			method: 'GET',
+			headers: {
+				'Content-Type': 'application/json',
+				Authorization: `Bearer ${token}`
+			}
+		})
+		return await response.json()
+	} catch (error) {
+		setTimeout(() => {
+			PopUpManager.addPopUp(
+				`Error fetching jars data: ${error.message}`,
+				'error'
+			)
+		}, 100)
 
-	const response = await fetch(endpoint, {
-		method: 'GET',
-		headers: {
-			'Content-Type': 'application/json',
-			Authorization: `Bearer ${token}`
-		}
-	})
-	const data = await response.json()
-
-	if (!response.ok) {
 		return null
 	}
-	return data
 }
 
-export const CardsList = () => {
-	const token = useRouteLoaderData('token')
-	const [cardData, setCardData] = useState(null)
+const cardType = {
+	platinum: 'platinum',
+	black: 'black',
+	eAid: 'e-dopomoga',
+	white: 'white'
+}
 
-	useEffect(() => {
-		const fetchData = async function () {
-			try {
-				const access = await getCards(token)
-				setCardData(access)
-			} catch (error) {
-				console.error('Error fetching card data:', error)
-			}
-		}
+export const CardsList = () => {
+	const cardData = useRouteLoaderData('cards')
 
-		let ignore = fetchData()
-	}, [token])
+	return (
+		<div className='cards'>
+			<h2 className='cards__title'>Credit Card List</h2>
```

Comment:
Lets pick same style for jars and cards. Now we have h1 and h2 headers with same priority pages

Reply:
Done